tests/fixtures/html/00731-v1-vor-section-author-info-funding.html

```diff
@@ -0,0 +1 @@
+<ul class="funding-group"><li><span class="funding-source"><span class="institution">European Research Council</span></span><span class="principal-award-recipient"><span class="name">Kentaro Yoshida</span><span class="name">Verena J Schuenemann</span><span class="name">Liliana M Cano</span><span class="name">Marina Pais</span><span class="name">Sophien Kamoun</span><span class="name">Johannes Krause</span></span></li><li><span class="funding-source"><span class="institution">Gatsby Charitable Foundation</span></span><span class="principal-award-recipient"><span class="name">Kentaro Yoshida</span><span class="name">Liliana M Cano</span><span class="name">Marina Pais</span><span class="name">Sophien Kamoun</span></span></li><li><span class="funding-source"><span class="institution">Biotechnology and Biological Sciences Research Council (BBSRC)</span></span><span class="principal-award-recipient"><span class="name">Kentaro Yoshida</span><span class="name">Liliana M Cano</span><span class="name">Marina Pais</span><span class="name">Sophien Kamoun</span></span></li><li><span class="funding-source"><span class="institution">LOEWE</span></span><span class="principal-award-recipient"><span class="name">Bagdevi Mishra</span><span class="name">Rahul Sharma</span><span class="name">Marco Thines</span></span></li><li><span class="funding-source"><span class="institution">United States Department of Agriculture</span></span><span class="principal-award-recipient"><span class="name">Frank N Martin</span></span></li><li><span class="funding-source"><span class="institution">Japan Society for the Promotion of Science</span></span><span class="principal-award-recipient"><span class="name">Kentaro Yoshida</span></span></li><li><span class="funding-source"><span class="institution">Deutsche Forschungsgemeinschaft</span></span><span class="principal-award-recipient"><span class="name">Detlef Weigel</span><span class="name">Hernán A Burbano</span></span></li><li><span class="funding-source"><span class="institution">Max Planck Society</span></span><span class="principal-award-recipient"><span class="name">Chirsta Lanz</span><span class="name">Detlef Weigel</span><span class="name">Hernán A Burbano</span></span></li></ul><p class="funding-statement">The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</p>
```

Comment:
Yeah, don't think this adds anything. We have a ton of target="_blank" in references already. Unless this brings any extra value then leave ti out because it gives us more fixtures to maintain if we have to refactor markup.